nav2_docking/opennav_docking/src/docking_server.cpp

```diff
@@ -744,6 +760,13 @@ DockingServer::dynamicParametersCallback(std::vector<rclcpp::Parameter> paramete
       if (name == "max_retries") {
         max_retries_ = parameter.as_int();
       }
+    } else if(type == ParameterType::PARAMETER_BOOL){
```

Comment:
```suggestion
    } else if (type == ParameterType::PARAMETER_BOOL) {
```

You seem to have more linting issues - I'm not sure why CI didn't catch them. Please run `ament_cpplint` and `ament_uncrustify`

Reply:
Oh, CI did pick up on them: https://app.circleci.com/pipelines/github/ros-navigation/navigation2/12898/workflows/0f40ab36-340c-4242-befb-e9c0abdfdc04/jobs/38897/tests